### DISCUSSION TITLE
Why is my account suspended?

### DISCUSSION THREAD

**Discussion by IluminatiX-II** [community/community#120413] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello. I'm sorry to bother you about this, knowing that it won't help my situation. But I would like to understand what happened to me...

A few days ago when I woke up in the morning, I lost access to my account and was told that my account was suspended after violating Github rules. I was instantly surprised, because even though I use github a lot, I have absolutely no idea what I did to get my account suspended. What's more, I received no warning message, absolutely nothing to give me any details as to why. And I don't even appear on github any more, as if my account had become a ghost... I sent a message with a ticket to ask for an explanation but I hope I'll get an answer soon. I'm the owner of several translations from the Paratranz site that go through github and I no longer have access to anything because of this problem...

That's why I'm writing this message with my second account to find out if anyone can advise me what to do? or if there's nothing I can do apart from wait without knowing why I've been suspended...

**Comment by RM25BAM** [comment]
Hi @IluminatiX-II ! That's horrible what happened to you. I would recommend you to connect with [Github Support](https://support.github.com/request). I hope this helps, good luck!

**Reply by IluminatiX-II** [reply]
that's what I did... I hope I'll get a quick answer...

**Reply by Naman-Acharya** [reply]
I don't know how but most of users faced similar issue even my account got suspended without any reason. Might be this is because of malware as per the current news but still unaware about it. I hope we get our account back as soon as possible because specially most of my work was on Github and with Github Copilot and all the work was suddenly stopped because of this incident. 

My Ticket No: #2737022

**Comment by Youssef-BS** [comment]
If you haven't received a response within a reasonable amount of time, consider following up with GitHub support.

**Comment by ebndev** [comment]
Hi @IluminatiX-II,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion and any duplicate discussions.

We appreciate your understanding!
